concent_api/verifier/utils.py

```diff
@@ -336,19 +336,31 @@ def unpack_archives(file_paths: Iterable[str], subtask_id: str) -> None:
             )
 
 
-def validate_downloaded_archives(subtask_id: str, archives_list: Iterable[str]) -> None:
-    # If any file which is supposed to be unpacked from archives already exists, finish with error and raise exception.
+def validate_downloaded_archives(subtask_id: str, archives_list: Iterable[str], scene_file: str) -> None:
+    # If archive is broken, it means that Provider must have intentionally uploaded damaged zip file.
+    # In such case verification end with MISMATCH result.
+    package_files_list = []  # type: List[str]
+    try:
+        # If any file which is supposed to be unpacked from archives already exists, finish with error and raise exception.
+        for package_file_path in archives_list:
+            package_files_list += get_files_list_from_archive(
```

Comment:
Wouldn't be better to use built-in `append()` function? :)

Reply:
why? `+=` seems more concise ;)